sdk/monitor/monitor-opentelemetry/src/index.ts

```diff
@@ -82,7 +82,7 @@ export function useAzureMonitor(options?: AzureMonitorOpenTelemetryOptions): voi
     metricReader: metricHandler.getMetricReader(),
     views: metricHandler.getViews(),
     instrumentations: instrumentations,
-    logRecordProcessor: logHandler.getAzureLogRecordProcessor(),
+    logRecordProcessors: [logHandler.getAzureLogRecordProcessor()],
```

Comment:
You should add the logRecordProcessors defined below, we added that one after because it was not supported by OTel

Reply:
Order is important as the last one should be the one with the exporter associated, also do we allow these processors to be passed as config? if we do, we need to respect that

Reply:
I see, cleaned up the old solution for logRecordProcessors and spanProcessors and just incorporated it all into the `sdkConfig`. From my testing order was respected before, and I made sure that we maintain that order with the new changes I've just added. (AzureLogRecordProcessor, user processors, batchLogRecordProcessor). 

I'm a little confused about your question regarding the processors being passed in as config. We do allow them to be passed in via the `AzureMonitorOpenTelemetryOptions` and the changes here respect those values by adding them to this array passed to `sdk.start()`.